src/Hl7.Fhir.Serialization/FhirXmlNode.cs

```diff
@@ -438,7 +438,37 @@ object checkRepresentation(ITypedElement node, IExceptionSource ies, object _)
         private ExceptionNotification buildException(string message) => ExceptionNotification.Error(
                 new StructuralTypeException("Parser: " + message));
 
-        public string OriginalElement => ((XElement)Current).ToString(SaveOptions.DisableFormatting);
+        public string OriginalElement
+        {
+            get {
+                if (((XElement)Current.Parent).Name.Namespace != ((XElement)Current).Name.Namespace)
```

Comment:
Null reference exception when you call this property on the root of a document, since there won't be a .Parent.

Reply:
Feels a bit like a hack?  If it is, we better add a bit of documentation here.